package.json

```diff
@@ -58,15 +58,6 @@
     "url-regex": "3.2.0",
     "uuid": "3.0.1"
   },
-  "peerDependencies": {
```

Comment:
nope! had to remove them temporarily in order to run `flow-typed` but forgot to put them back, thanks!